fuel-indexer-tests/src/lib.rs

```diff
@@ -14,6 +14,8 @@ pub mod defaults {
     pub const WEB_API_ADDR: &str = "127.0.0.1:8000";
     pub const PING_CONTRACT_ID: &str =
         "68518c3ba3768c863e0d945aa18249f9516d3aa1338083ba79467aa393de109c";
+    pub const TRANSFER_BASE_ASSET_ID: &str =
```

Comment:
- This is already exported by `fuels::prelude::BASE_ASSET_ID`.
- Can probably just import that directly instead of vial `fuel_indexer_tests::defaults`

Reply:
Oops, that was left over; I'll remove it in another PR.